data/transit/route/train.json

```diff
@@ -789,6 +789,17 @@
         "route": "train"
       }
     },
+    {
+      "displayName": "Romney, Hythe & Dymchurch Railway",
+      "locationSet": {"include": ["gb"]},
+      "tags": {
+        "network": "Romney, Hythe & Dymchurch Railway",
+        "network:wikidata": "Q2090360",
+        "network:wikipedia": "en:Romney, Hythe and Dymchurch Railway",
+        "operator": "Romney, Hythe & Dymchurch Railway PLC",
```

Comment:
Any reason for the "PLC" here?  Seems like it should just be the same thing as what `network` has.

Reply:
I just went off Wikipedia: https://en.wikipedia.org/wiki/Romney,_Hythe_and_Dymchurch_Railway#Ownership_and_operation

Reply:
Gotcha - I think I'm just going to put them as the same thing for now, and we can always change it later if the railway and the owning company ever end up having separate wikidata ids.